libraries/AP_HAL_ChibiOS/hwdef/JHEMCUF405WING/hwdef.dat

```diff
@@ -0,0 +1,180 @@
+# hw definition file for JHEMCU F4 WING hardware
+# tested on the JHEMCU F405 WING board
+
+# MCU class and specific type
+MCU STM32F4xx STM32F405xx
+
+# board ID for firmware load
+APJ_BOARD_ID 1160
+
+# crystal frequency
+OSCILLATOR_HZ 8000000
+
+define STM32_ST_USE_TIMER 5
+define CH_CFG_ST_RESOLUTION 32
+
+# reserve 16k for bootloader, 16k for OSD and 32k for flash storage
+FLASH_RESERVE_START_KB 64
+FLASH_SIZE_KB 1024
+
+define HAL_STORAGE_SIZE 15360
+STORAGE_FLASH_PAGE 1
+
+# only one I2C bus
+I2C_ORDER I2C1
+
+# order of UARTs (and USB)
+SERIAL_ORDER OTG1 USART1 USART2 USART3 UART4 UART5 USART6
+
+# LEDs
+PA13 LED_GREEN OUTPUT LOW GPIO(0)
+PA14 LED_BLUE OUTPUT LOW GPIO(1)
+
+define HAL_GPIO_A_LED_PIN 0
+define HAL_GPIO_B_LED_PIN 1
+
+# buzzer
+PC15 BUZZER OUTPUT GPIO(80) LOW
+define HAL_BUZZER_PIN 80
+
+# spi1 bus for IMU
+PA5 SPI1_SCK SPI1
+PA6 SPI1_MISO SPI1
+PA7 SPI1_MOSI SPI1
+PA4 MPU_CS CS
+
+# spi2 for OSD
+PB13 SPI2_SCK SPI2
+PC2  SPI2_MISO SPI2
+PC3  SPI2_MOSI SPI2
+PB12 OSD_CS CS
+
+# spi3 for  sdcard
+PB3 SPI3_SCK SPI3
+PB4 SPI3_MISO SPI3
+PB5 SPI3_MOSI SPI3
+PC14 SDCARD_CS CS
+
+# only one I2C bus in normal config
+PB8 I2C1_SCL I2C1
+PB9 I2C1_SDA I2C1
+
+# analog pins
+PC0 BATT_VOLTAGE_SENS ADC1 SCALE(1)
+PC1 BATT_CURRENT_SENS ADC1 SCALE(1)
+PC4 RSSI_ADC_PIN      ADC1 SCALE(1)
+
+PC5 PRESSURE_SENS     ADC1 SCALE(1)
+define HAL_DEFAULT_AIRSPEED_PIN 15
+
+# define default battery setup
+define HAL_BATT_MONITOR_DEFAULT 4
+define HAL_BATT_VOLT_PIN 10
+define HAL_BATT_CURR_PIN 11
+define HAL_BATT_VOLT_SCALE 11.05 # matched to PDB board
+define HAL_BATT_CURR_SCALE 50    # matched to PDB board
+
+# analog rssi pin
+define BOARD_RSSI_ANA_PIN 14
+
+# USART1 (ELRS)
+PA9 USART1_TX USART1
+PA10 USART1_RX USART1
+
```

Comment:
```suggestion
define DEFAULT_SERIAL1_PROTOCOL SerialProtocol_RCIN
```
Since this port is shown as the connection point for RC like CRSF/ELRS...it does not interfere with the SBUS input to UART2, either can be connected